src/Microsoft.Identity.Web.GraphServiceClient/Microsoft.Identity.Web.GraphServiceClient.csproj

```diff
@@ -17,7 +17,12 @@
     <PackageReference Include="System.Diagnostics.DiagnosticSource" Version="8.0.0" />
   </ItemGroup>
 
-  <ItemGroup Condition="'$(TargetFramework)' != 'net8.0'">
+  <ItemGroup Condition="'$(TargetFramework)' == 'net9.0'">
+    <PackageReference Include="Microsoft.Graph" Version="5.40.0" />
```

Comment:
Why not always 5.40.0? is there an incompatiblity?

Reply:
Not for Microsoft.Graph from net8 to net9 but there's incompatibility in System.Diagnostics.DiagnosticSource as for why net7 and below are different that's something that was probably found in the NET8 work so, I just followed the pattern.

Reply:
@jennyf19 shouldn't we move to Graph 5.40.0 for all FW?

Reply:
Can we open a work item and do it as a separate PR?

Reply:
Create issue to follow up https://github.com/AzureAD/microsoft-identity-web/issues/2748. Thanks @jmprieur & @jennyf19